pkg/api/login_oauth.go

```diff
@@ -17,9 +23,9 @@ import (
 )
 
 func GenStateString() string {
-        rnd := make([]byte, 32)
-        rand.Read(rnd)
-        return base64.StdEncoding.EncodeToString(rnd)
+	rnd := make([]byte, 32)
```

Comment:
I'm not sure about formal guidelines, but all the existing code I've looked at (and in this case the rest of the file) uses tabs.